pkg/services/ngalert/notifier/channels/line.go

```diff
@@ -95,6 +98,19 @@ func (ln *LineNotifier) Notify(ctx context.Context, as ...*types.Alert) (bool, e
 	form := url.Values{}
 	form.Add("message", body)
 
+	_ = withStoredImages(ctx, ln.log, ln.images,
+		func(index int, image *ngmodels.Image) error {
+			// If there is an image for this alert and the image has been uploaded
+			// to a public URL then add it to the request. We cannot add more than
+			// one image per request.
+			if image != nil && image.URL != "" && !form.Has("imageThumbnail") && !form.Has("imageFullsize") {
+				form.Add("imageThumbnail", image.URL)
```

Comment:
The maximum size of this image thumbnail is 240x240px. Have you been able to test whether the thumbnail successfully appears with a larger image? It may be fine to just leave the thumbnail out, depending on what the message looks like with only `imageFullsize` set.